### PR TITLE
Improve package.json

### PR DESCRIPTION
**Pull request by danji90**
Improved metadata description in the package.json file for mobility-toolbox-js to increase our "Package Score".
See https://docs.skypack.dev/package-authors/package-checks for details.

# Others

- [ ] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [ ] Everything in ticket description has been fixed.
- [ ] The author of the MR has made its own review before assigning the reviewer.
- [ ] The title means something for a human being.
- [ ] The title contains [WIP] if it's necessary.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] The new class' members & methods are well documented
- [ ] Tests added.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -154,6 +155,17 @@
...
+    ".": "./index.js",
+    "./module": "./module.js",
+    "./api": "./api/index.js",
+    "./common": "./common/Tracker.js",
+    "./common/layers": "./common/layers/layer.js",
+    "./common/mixins": "./common/mixins/*.js",
+    "./mapbox": "./mapbox/index.js",
+    "./ol": "./ol/index.js",
+    "./ol/Map": "./ol/Map.js"
+  },
```
Comment (danji90) [inline package.json:168]: I think I included all the export paths, but please verify if they are all there

Comment (oterral) [reply package.json:168]: I don't know if we have to put every of them .
I think . , module, api, common,mapbox and ol are the most important, the rest isn't

```
".": "./index.js",
￼    "./module": "./module.js",
￼    "./api": "./api/index.js",
￼    "./common": "./common/*.js",
￼    "./mapbox": "./mapbox/index.js",
￼    "./ol": "./ol/index.js",
```

Comment (danji90) [reply package.json]: @oterral Fixed. LGTM?

Comment (oterral) [reply package.json]: yes

**Pull request merged**